db/institutions.yml

```diff
@@ -793,6 +833,7 @@ ufrankfurt:
     - Frankfurt University
     - Goethe-Universität
     - Universität Frankfurt am Main
+    - Johann Wolfgang Goethe-Universität
```

Comment:
ah, nice.  please change the reference in people to the id ufrankfurt, not what i wrote in the other PR